benchmarks/bench_methods_run.py

```diff
@@ -220,28 +183,17 @@ def one_run(X, X_shape, random_state, method, n_atoms, n_times_atom, reg):
 
     if len(X_shape) == 2:
         n_trials, n_times = X.shape
-        n_channels = 1
-        X_init = X[:, None, :]
     else:
         n_trials, n_channels, n_times = X.shape
-        X_init = X
-
-    # use the same init for all methods
-    ds_init = init_dictionary(X_init, n_atoms, n_times_atom, D_init='chunk',
-                              rank1=False, uv_constraint='separate',
-                              random_state=random_state)
-    if len(X_shape) == 2:
-        ds_init = ds_init[:, 0, :]
```

Comment:
Quick question is: are you sure the init is the same for all the methods (same random_state)?
If so, works for me.

Reply:
Yes, at line 250 random_state is set `iterator = itertools.product(methods, range(n_states))`. 
All methods use the same random_state and I checked the dict to see if the values are the same.